docs/manual/docs/overview/authors.md

```diff
@@ -1,54 +1,118 @@
--   Francois Prunayre
--   Jesse Eichar
--   Andrea Carboni (acarboni * crisalis-tech com - Independent consultant)
--   Archie Warnock (warnock * awcubed com) [A/WWW Enterprises](https://www.awcubed.com)
-
-## Committers
-
-Committers list is available [here for the core team](https://github.com/orgs/geonetwork/teams/committers) and [here for the documenter team](https://github.com/orgs/geonetwork/teams/documenters).
-
--   Emanuele Tajariol
--   Craig Jones
--   Florent Gravin
--   Francois Prunayre
--   Jeroen Ticheler
--   Jose Garcia
--   Maria Arias de Reyna
--   Juan Luis Rodriguez Ponce
--   Olivier Guyot
--   Pierre Mauduit
--   Simon Pigot
--   Paul van Genuchten
--   Antonio Cerciello
--   Michel Gabriel
--   Jody Garnett
--   David Blasby
+GeoNetwork was started in 2001 as project of the Food and Agriculture Organization of the United Nations (FAO-UN), United Nations World Food Programme (WFP) and United Nations Environment Programme (UNEP).
+
+The open-source project was initially managed by Jeroen Ticheler and then donated to the Open Source Geospatial Foundation. GeoNetwork [graduated](https://www.osgeo.org/foundation-news/geonetwork-opensource-graduates-osgeo-incubation/) as full Open Source Geospatial project in 2008 and has been managed by a project steering committee and active committers in subsequent years.
+
+## Project Leadership
+
+The GeoNetwork project roadmap is managed using a proposal process, with the GeoNetwork leadership team voting on proposals made by members of our community. This process has moved from mailing list to a [project board](https://github.com/orgs/geonetwork/projects/2).
+
+Active developers in GeoNetwork project have taken on responsibility for reviewing community contributions, making releases, handling security vulnerability reports.
+
+This dedication is reflected in community trust, and a Leadership responsibility to vote on roadmap proposals:
+
+* [David Blasby](https://github.com/davidblasby): [Active Maintainer](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Adavidblasby)
+* [Jose García](https://github.com/josegar74): [Active Maintainer](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Ajosegar74)
+* [Jody Garnett](https://github.com/jodygarnett): [Active Maintainer](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Ajodygarnett)
+* [François Prunayre](https://github.com/fxprunayre): [Active Maintainer](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed++reviewed-by%3Afxprunayre)
+* [Juan Luis Rodríguez Ponce](https://github.com/juanluisrp): [Active Maintainer](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Ajuanluisrp)
+* [Ian Wallen](https://github.com/ianwallen): [Active Maintainer](https://github.com/geonetwork/core-geonetwork/pulls?q=is%3Apr+is%3Aclosed+reviewed-by%3Aianwallen)
```

Comment:
Thank you for the nomination however I don't have any extra time to contribute - Sorry, I will have to decline

```suggestion

```